src/main/java/ru/hogwards/school/mapper/FacultyMapper.java

```diff
@@ -0,0 +1,26 @@
+package ru.hogwards.school.mapper;
+
+import org.springframework.stereotype.Component;
+import ru.hogwards.school.dto.FacultyDtoIn;
+import ru.hogwards.school.dto.FacultyDtoOut;
+import ru.hogwards.school.dto.StudentDtoIn;
+import ru.hogwards.school.dto.StudentDtoOut;
+import ru.hogwards.school.entity.Faculty;
+import ru.hogwards.school.entity.Student;
+
+@Component
+public class FacultyMapper {
+    public FacultyDtoOut toDto(Faculty faculty){
+            FacultyDtoOut facultyDtoOut = new FacultyDtoOut();
+            facultyDtoOut.setId(faculty.getId());
+            faculty.setName(faculty.getName());
+            faculty.setColour(faculty.getColour());
```

Comment:
```suggestion
        facultyDtoOut.setName(faculty.getName());
        facultyDtoOut.setColour(faculty.getColour());
```
Обрати внимание, что заполняешь не тут объект